pkg/scm/scm.go

```diff
@@ -45,8 +45,7 @@ func DownloaderForSource(fs util.FileSystem, s string, forceCopy bool) (build.Do
 		return &file.File{FileSystem: fs}, s, nil
 	}
 
-	// If the source is valid  Git protocol (file://, ssh://, git://, git@, etc..) use Git
```

Comment:
nit, but I personally preferred the comment as it was - the new comment might as well not exist at all as it can be inferred from the code below; the old one at least brought to mind some of the potential options the code might be dealing with.